doc/tutorials/imgproc/pyramids/pyramids.markdown

```diff
@@ -105,15 +105,15 @@ Let's check the general structure of the program:
     while( true )
     {
       int c;
-      c = waitKey(10);
+      c = waitKey(10) & 0xFF;
 
-      if( (char)c == 27 )
+      if( c == 27 )
```

Comment:
But why would this generate a warning? Buildbot builds samples and there is not a single one out there. If c is an int, then comparing it to 27 or to any other numerical value should work just fine in all C++ standards. Could you elaborate where this could go wrong?